From f6abee788c304ee7616e699cf52e85a4a7b73746 Mon Sep 17 00:00:00 2001
From: Catalin Horghidan <catalin.horghidan@nxp.com>
Date: Mon, 1 Oct 2018 13:57:25 +0300
Subject: [PATCH 308/706] felix: dts: add phy nodes for RDB

Signed-off-by: Catalin Horghidan <catalin.horghidan@nxp.com>
(cherry picked from commit 2f3af866efd1cabe2c54a9141bae69fa22bc1d9c)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 .../boot/dts/freescale/fsl-ls1028a-rdb.dts    | 41 +++++++++++++++++++
 1 file changed, 41 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a-rdb.dts b/arch/arm64/boot/dts/freescale/fsl-ls1028a-rdb.dts
index 996dd257efae..5d4fb2ef3e9f 100755
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a-rdb.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a-rdb.dts
@@ -233,6 +233,26 @@
 		compatible = "ethernet-phy-id004d.d074";
 		reg = <0x2>;
 	};
+
+	qsgmii_phy1: ethernet-phy@4 {
+		compatible = "ethernet-phy-id0007.0670";
+		reg = <0x4>;
+	};
+
+	qsgmii_phy2: ethernet-phy@5 {
+		compatible = "ethernet-phy-id0007.0670";
+		reg = <0x5>;
+	};
+
+	qsgmii_phy3: ethernet-phy@6 {
+		compatible = "ethernet-phy-id0007.0670";
+		reg = <0x6>;
+	};
+
+	qsgmii_phy4: ethernet-phy@7 {
+		compatible = "ethernet-phy-id0007.0670";
+		reg = <0x7>;
+	};
 };
 
 &enetc_port0 {
@@ -259,3 +279,24 @@
 		full-duplex;
 	};
 };
+
+/* l2switch ports */
+&switch_port0 {
+	phy-handle = <&qsgmii_phy1>;
+	phy-connection-type = "qsgmii";
+};
+
+&switch_port1 {
+	phy-handle = <&qsgmii_phy2>;
+	phy-connection-type = "qsgmii";
+};
+
+&switch_port2 {
+	phy-handle = <&qsgmii_phy3>;
+	phy-connection-type = "qsgmii";
+};
+
+&switch_port3 {
+	phy-handle = <&qsgmii_phy3>;
+	phy-connection-type = "qsgmii";
+};
-- 
2.17.1

